keras_cv/models/__internal__/unet.py

```diff
@@ -0,0 +1,172 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from tensorflow.keras import initializers
+from tensorflow.keras import layers
+from tensorflow.keras import regularizers
+
+
+def Block(filters, downsample):
```

Comment:
Consider adding docstring (even though it's not public API)

Reply:
done